samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,351 @@
+   The sample will wait this many seconds for the modem to determine the current date and time before giving up and moving on.
+
+.. _CONFIG_APPLICATION_THREAD_STACK_SIZE:
+
+CONFIG_APPLICATION_THREAD_STACK_SIZE - Application Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the application thread of the sample.
+
+.. _CONFIG_CONNECTION_THREAD_STACK_SIZE:
+
+CONFIG_CONNECTION_THREAD_STACK_SIZE - Connection Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the connection thread of the sample.
+
+.. _CONFIG_MESSAGE_THREAD_STACK_SIZE:
+
+CONFIG_MESSAGE_THREAD_STACK_SIZE - Message Queue Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the message queue processing thread of the sample.
+
+.. _CONFIG_MAX_OUTGOING_MESSAGES:
+
+CONFIG_MAX_OUTGOING_MESSAGES - Outgoing message maximum
+   Sets the maximum number of messages that can be in the outgoing message queue.
+   Messages submitted past this limit will not be enqueued.
+
+.. _CONFIG_MAX_CONSECUTIVE_SEND_FAILURES:
+
+CONFIG_MAX_CONSECUTIVE_SEND_FAILURES - Max outgoing consecutive send failures
+   Sets the maximum number of device messages which may fail to send before a connection reset and cooldown is triggered.
+
+.. _CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS:
+
+CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS - Cooldown after max consecutive send failures exceeded (seconds)
+   If a connection reset is triggered by too many failed device messages, the sample will wait this long (in seconds) before trying again.
+
+.. _CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS:
+
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS - Sensor sampling interval (seconds)
+   Sets the time to wait between each temperature sensor sample.
+
+.. warning::
+   Every time a sensor sample is sent, it will interrupt any attempted GNSS fix.
+   If you set this interval too short, there will not be enough time to ever acquire a fix.
+   In good conditions, and with AGPS data, one minute is generally just barely long enough to get a fix most of the time.
+
+.. _CONFIG_GNSS_ANTENNA_ONBOARD:
+
+CONFIG_GNSS_ANTENNA_ONBOARD - Select the onboard GNSS antenna (default)
+   This option enables the onboard GNSS antenna.
+
+
+.. _CONFIG_GNSS_ANTENNA_EXTERNAL:
+
+CONFIG_GNSS_ANTENNA_EXTERNAL - Select an external GNSS antenna
+   This option enables the external GNSS antenna.
+
+
+.._CONFIG_GNSS_AT_MAGPIO:
+
+CONFIG_GNSS_AT_MAGPIO - ``AT%XMAGPIO`` command
+   Sets the ``AT%XMAGPIO`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
+
+.._CONFIG_GNSS_AT_COEX0:
+
+CONFIG_GNSS_AT_COEX0 - ``AT%XCOEX0`` command
+   Sets the ``AT%XCOEX0`` command to be sent to GNSS module. Leave empty if this command should not be sent. This command must be sent for GNSS to function properly. See `nRF9160 SiP pin configuration`_ for more details.
```

Comment:
In the context of this sample, I don't think the user would need to mess with these two config items. They get correct values based on the board and antenna configuration. Therefore I think it might be confusing if they are mentioned here. Or if you want to keep them (for the sake of completeness) maybe you could alter the comment to state what I just wrote.

Reply:
I agree, although I do wish to keep them in the documentation for completeness, since they are required for the sample to operate correctly.

I will do as you suggest, and alter the comment.

The body now reads:
```

   Sets the ``AT%XMAGPIO`` command to be sent to the GNSS module.
   Leave empty if this command should not be sent.
   This command must be sent for GNSS to function properly.

   A reasonable value (that selects the internal GNSS antenna) is automatically selected for this option based on the build target.
   You should never have to mess with it yourself unless you are using a non-standard board or custom GNSS antenna.
   See `nRF9160 SiP pin configuration`_ for more details.
  ```